packages/terra-action-header/src/ActionHeader.scss

```diff
@@ -0,0 +1,15 @@
+local {
```

Comment:
Shouldn't this be " :local { " to make the styles local to this file?

Reply:
These styles weren't being used anyways. Removed [here](https://github.com/cerner/terra-core/commit/728d4f722bd7854c04324e910d74905c72737db1).